libkpod/kill.go

```diff
@@ -0,0 +1,34 @@
+package libkpod
+
+import (
+	"github.com/docker/docker/pkg/signal"
+	"github.com/kubernetes-incubator/cri-o/oci"
+	"github.com/kubernetes-incubator/cri-o/utils"
+	"github.com/pkg/errors"
+	"os"
+)
+
+// ContainerKill sends the user provided signal to the containers primary process.
+func (c *ContainerServer) ContainerKill(container string, signalString string) (string, error) {
```

Comment:
I don't like passing a string for the signal - can you parse it in cmd/kpod and pass an it here as an int?

Reply:
./vendor/github.com/docker/docker/pkg/signal/signal_linux.go
Has a translation table.